scylla/src/transport/load_balancing/default.rs

```diff
@@ -673,7 +673,11 @@ impl DefaultPolicyBuilder {
         // alter the `DefaultPolicyBuilder`'s API so that it is impossible for the user to create such state.
         let preferences = match (self.preferred_datacenter, self.preferred_rack) {
             (None, None) => ReplicaLocationPreference::Any,
-            (None, Some(_)) => ReplicaLocationPreference::Any, // this is case that the user shouldn't be able to represent
+            (None, Some(_)) => {
+                // This a is case that the user shouldn't be able to represent.
+                warn!("Preferred rack has effect only if a preferred datacenter is set as well. Ignoring the preferred rack.");
```

Comment:
Nit: this code does not belong to its commit, it included in a commit that was supposed to only update the documentation.